src/app/(main)/EventPage.tsx

```diff
@@ -63,7 +76,20 @@ const Roster = ({participants, orgs, startTime}: {participants: Record<string, P
       hideFooter
       rowSelection={false}
       onRowClick={handleRowClick}
-      getRowHeight={() => 'auto'}
+      getRowHeight={({ densityFactor }: GridRowHeightParams) => {
+        return 52 * densityFactor;

```

Comment:
What is magic about 52? Also I still don't understand what units these heights and widths are in. Is it pixels?

Reply:
Per the MUI documentation 52 is the default height of the data grid rows.

Reply:
I think it's pixels. The default line height is 48.